frontend/src/metabase/xray/xray.js

```diff
@@ -10,57 +10,26 @@ import {
 
 import { XRayApi } from 'metabase/services'
 
-export const FETCH_FIELD_XRAY = 'metabase/xray/FETCH_FIELD_XRAY'
-export const fetchFieldXray = createThunkAction(FETCH_FIELD_XRAY, (fieldId, cost) =>
+export const FETCH_XRAY = 'metabase/xray/FETCH_XRAY'
+export const fetchXray = createThunkAction(
```

Comment:
Personally I would maybe like to see separate methods for constructing the `fetchXray(type, id, cost)` method call, maybe even separate actions for each of those (that would in turn dispatch fetchXray). Currently the API is maybe a little unclear to me, at least if I only look at the actions in `xray.js`.